lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -225,10 +229,31 @@ sub create_from_settings {
     return $job;
 }
 
+sub search_modules ($self, $module_re) {
+    my $distris = path(testcasedir);
+    my @results;
+    for my $distri ($distris->list({dir => 1})->map('realpath')->uniq()->each) {
+        next unless -d $distri;
+
+        my @cmd = ('git', '-C', $distri, 'grep', '-l', $module_re, '--', '*.p[my]');
+        my $stdout;
+        my $stderr;
+        IPC::Run::run(\@cmd, \undef, \$stdout, \$stderr);
+        next if $stderr;
```

Comment:
Do we want to know if the grep failed? We should probably check for the exit code then. It's in `$?`.
E.g. try with the input `foo[z-a]`

Reply:
`Invalid range end` ... but still I don't think this is interesting for user